astro_2.0/features/Feed/Feed.module.scss

```diff
@@ -16,7 +16,7 @@
   @include wide-background;
   padding: 56px 0 0;
   position: sticky;
-  top: 64px;
+  top: 72px;
```

Comment:
We should adjust it with proposal type filter.

Reply:
done